### ISSUE TITLE
mount: some mount options are not worked.

### ISSUE THREAD

**Issue by biubiuzy** [util-linux/util-linux#3256]
1. It seems that the lazytime option did not work.
```
# mount -o lazytime /dev/loop0 /tmp/mnt
# mount| grep loop0
/dev/loop0 on /tmp/mnt type ext4 (rw,relatime)
```

The old mount(2) syscall works as expected:
```
# LIBMOUNT_FORCE_MOUNT2=always mount -o lazytime /dev/loop0 /tmp/mnt
# mount| grep loop0
/dev/loop0 on /tmp/mnt type ext4 (rw,relatime,lazytime)
```

**Comment by karelzak** [comment]
Version? It should be fixed in v2.40.

The current tree (and kernel 6.12.0-0.rc4):
```
strace -e fsconfig,fsopen,fspick,open_tree,move_mount,mount_setattr ./mount -o lazytime /dev/sdc /mnt/test
...
fsopen("ext4", FSOPEN_CLOEXEC)          = 3
fsconfig(3, FSCONFIG_SET_STRING, "source", "/dev/sdc", 0) = 0
fsconfig(3, FSCONFIG_SET_FLAG, "lazytime", NULL, 0) = 0
fsconfig(3, FSCONFIG_CMD_CREATE, NULL, NULL, 0) = 0
move_mount(4, "", AT_FDCWD, "/mnt/test", MOVE_MOUNT_F_EMPTY_PATH) = 0
```

and result:
```
findmnt /mnt/test
TARGET    SOURCE   FSTYPE OPTIONS
/mnt/test /dev/sdc ext4   rw,relatime,lazytime,stripe=512
```

**Comment by biubiuzy** [comment]
54e4a6b145fd6ef943d93e16de748283e687855d Fixed，close.
